src/optimizer/child_property_deriver.cpp

```diff
@@ -78,7 +81,7 @@ void ChildPropertyDeriver::Visit(UNUSED_ATTRIBUTE const HashGroupBy *op) {
 
 void ChildPropertyDeriver::Visit(UNUSED_ATTRIBUTE const SortGroupBy *op) {
```

Comment:
I know you didn't really make changes to this, but should we remove `UNUSED_ATTRIBUTE` here because we are using `op`

Reply:
Yeah, updated